examples/mediatek/executor_runner/mtk_llama_runner.h

```diff
@@ -43,11 +43,9 @@ class MTKLlamaRunner : public executorch::extension::llm::IRunner {
   Error load();
   Error generate(
       const std::string& prompt,
-      int32_t seq_len = 128,
+      executorch::extension::llm::GenerationConfig config,
```

Comment:
Can we leave mtk code as is? It's currently in unstable stage given the new apk release and upcoming code change.